go.mod

```diff
@@ -7,8 +7,9 @@ require (
 	github.com/Depado/ginprom v1.2.1-0.20200115153638-53bbba851bd8
 	github.com/araddon/dateparse v0.0.0-20190622164848-0fb0a474d195
 	github.com/aristanetworks/goarista v0.0.0-20190204200901-2166578f3448 // indirect
-	github.com/asaskevich/govalidator v0.0.0-20190424111038-f61b66f89f4a
 	github.com/avast/retry-go v2.6.0+incompatible
+
+	// TODO: Can this be removed?
```

Comment:
Did you mean to leave these TODOs in the PR?

Reply:
Yup I think @topliceanu is gonna continue the work